net.xmind.XMind.json

```diff
@@ -0,0 +1,74 @@
+{
+  "id": "net.xmind.XMind",
+  "command": "xmind",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "21.08",
+  "sdk": "org.freedesktop.Sdk",
+  "base": "org.electronjs.Electron2.BaseApp",
+  "base-version": "21.08",
+  "separate-locales": false,
+  "rename-icon":"xmind",
+  "finish-args": [
+    "--socket=x11",
+    "--share=ipc",
+    "--device=dri",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--filesystem=xdg-download",
+    "--filesystem=xdg-documents",
+    "--filesystem=xdg-pictures",
+    "--filesystem=xdg-desktop",
+    "--filesystem=/tmp",
+    "--talk-name=org.freedesktop.Notifications"
+  ],
+  "modules": [
+    {
+      "name": "xmind",
+      "buildsystem": "simple",
```

Comment:
You mention it's an "official package" and marked the related checkbox, but how can I verify that the company knows about this submission?

Reply:
https://github.com/orgs/xmindltd/people

Reply:
Ah, excellent, thank you.